fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+      bcc:
+        type: string
+        description: Optional array of email addresses to bcc. These must correspond to valid staff account email addresses.
+      subject:
+        type: string
+        description: The subject of the email that will be sent. When not provided, defaults to the Draft Order Invoice template's subject, as specified in the store's Notification settings.
+      custom_message:
+        type: string
+        description: An optional custom message provided in the body of the email.
+  checkout:
+    type: object
+    description: Object describing the checkout for the draft order.
+  line_item:
+    type: object
+    discriminator: custom
+   
+    properties:
+      custom:
+        description: Indicates if this is product_variant_line_item, or a custom_line_item.  If set to `true` indicates that the `line_item` is a custom line item.  If set to `false` indicates that it is a product variant `line_item`. *Read only field*
+        type: boolean
+        enum: 
+        - true
+        - false
+        default: false
+        
+      id:
+          type: string
+          description: The `id` of the line item.
+          readOnly: true
+          
+      variant_id:
+          type: string
+          description: The `id` of the product variant corresponding to the `line_item`.  Null if custom `line_item`.
+          readOnly: false
+          
+      product_id:
+          type: string
+          description: The `id` of the product corresponding to the line item’s product variant.
+          readOnly: true
+      name:
+        type: string
+        description: The name of the product.
+        readOnly: true
+        
+      variant_title:
+        type: string
+        description: The title of the product variant.  Defaults to "Custom" for custom line items created via the API.
+        readOnly: true
+    
+          
+     
+      vendor:
+          type: string
+          description: vendor
+          readOnly: true
+      quantity:
+          type: string
+          description: quantity
+          readOnly: false
+      gift_card:
+          type: boolean
+          description: Indicates if the product is a gift card. Values - `true` or `false`.
+          readOnly: true
+      fulfillment_service:
+          type: string
+          description: Service provider who is doing the fulfillment. Valid values are either `manual` or the name of the provider. eg `amazon`, `shipwire`, etc.  Defaults to `manual` for custom line items.
+          readOnly: true
+      properties:
+          type: string
+          description: An array of custom information for an item that has been added to the cart. Often used to provide product customization options. For more information, see the [documentation on collecting customization information on the product page](https://help.shopify.com/themes/customization/products/get-customization-information-for-products).
```

Comment:
Not actually true since draft orders have nothing to do with carts. This should be added to the draft order, not cart.